src/Components/Facility/ConsultationDetails/index.tsx

```diff
@@ -452,7 +458,11 @@ export const ConsultationDetails = (props: any) => {
                     <Link
                       key={p.text}
                       className={tabButtonClasses(tab === p.text)}
-                      href={`/facility/${facilityId}/patient/${patientId}/consultation/${consultationId}/${p.text.toLocaleLowerCase()}`}
+                      href={`/facility/${
+                        patientData?.facility_object?.id
+                      }/patient/${
+                        patientData?.id
+                      }/consultation/${consultationId}/${p.text.toLocaleLowerCase()}`}
```

Comment:
Why were these modified? Anyways the `facilityId`/`patientId` would end up being correct since it'll redirect if facility/patient ID is wrong right?